lib/apiservers/engine/backends/volume.go

```diff
@@ -198,7 +202,7 @@ func (v *Volume) VolumeRm(name string) error {
 			return derr.NewRequestNotFoundError(fmt.Errorf("Get %s: no such volume", name))
 
 		case *storage.RemoveVolumeConflict:
-			return derr.NewRequestConflictError(fmt.Errorf("Volume '%s' is in use", name))
+			return derr.NewRequestConflictError(fmt.Errorf(err.Payload.Message))
```

Comment:
Because the message returned by the PL includes the container ID of the container using the image.  Squashing the error and returning the generic error above lacks detail the PL error has.